src/screens/Learning.js

```diff
@@ -34,11 +42,23 @@ export default ({
   const [disableAllOptions, setDisableAllOptions] = useState(false);
   const [shouldReshuffle, setshouldReshuffle] = useState(false);
 
+  const findLearntCategory = categories.find(cat =>
+    cat.phrasesIds.includes(currentPhrase?.id),
+  );
+
   useEffect(() => {
     setOriginalPhrases(categoryPhrases);
     setNewQuestionPhrase(categoryPhrases, categoryPhrases);
   }, [categoryPhrases]);
 
+  useEffect(() => {
+    setLearntPhrasesCategory(
```

Comment:
Why do you need this action ?

Reply:
learntPhrasesCategory is used only in this component so you can just use useState or maybe just const inside this component